docs/src/reference.md

```diff
@@ -24,7 +24,9 @@ joinpath
 open
 read
 read!
+readdlm
```

Comment:
I think you also need to add readdlm/writedlm to the "Container methods" section.

Reply:
done